Permissions_UITest/.vs/Permissions_UITest/xs/UserPrefs.xml

```diff
@@ -1,30 +1,35 @@
-﻿<Properties>
+﻿<Properties StartupConfiguration="{A3B4C280-9AE9-46B8-8484-848B3711DE81}|Unit Tests">
```

Comment:
what's purpose of this file? I have some feeling that it should be under gitignore

Reply:
I believe VS did this change